test/unit/testMemory.js

```diff
@@ -3,12 +3,12 @@ var proxyquire = require('proxyquire');
 var os = require('os');
 
 var mocks = {
-  './lib/sender': function() {
+  './lib/clients': function() {
     this.send = function() {};
   }
 };
 
-var metrics = proxyquire('index.js', mocks)({enabled: true});
+var metrics = proxyquire('index.js', mocks)({enabled: true, host: '127.0.0.1'});
```

Comment:
Is the host field here now necessary where it wasn't before? I assume it's fine without it, just wondering in the context of backwards compatibility.